src/routes/+layout.svelte

```diff
@@ -4,10 +4,14 @@
   import Seo from "../components/Seo.svelte"
   import Header from "../components/Header.svelte"
   import LocalizationProvider from "../providers/LocalizationProvider.svelte"
+  import { theme } from '../store/theme'
   import "../app.css"
 </script>
 
 <svelte:head>
+  <meta name="color-scheme" content={$theme === 'light' ? 'light' : $theme} />
```

Comment:
Is it not the same as that?
```suggestion
  <meta name="color-scheme" content={$theme} />
```

Reply:
Yes, it's the same. I removed the condition.